pkg/stub/handler.go

```diff
@@ -802,6 +828,25 @@ func (h *Handler) Handle(event v1.Event) error {
 					progressing.Reason = progressing.Reason + isName + " "
 				}
 			}
+			if len(progressing.Reason) > 0 {
+				progressing.LastUpdateTime = now
+				progressing.LastTransitionTime = now
+				logrus.Debugf("Handle changing processing from false to true")
+				progressing.Status = corev1.ConditionTrue
+			} else {
+				logrus.Debugln("there are no imagestreams availalble for import")
+				// see if there are unskipped templates, to set SamplesExists to true
+				if !cfg.ConditionTrue(v1.SamplesExist) {
+					if len(h.templateFile) > len(h.skippedTemplates) {
```

Comment:
this looks questionable?  isn't it possible that "h.skippedTemplates" contains names that aren't actually template names, so h.skippedTemplates could be larger than h.templateFile, even though there are still some template names that are not in the skip list?

Reply:
yep we don't validate the names put in the skipped field

switched to cross referencing the list of files against the skipped list